modules/dnn/src/torch/torch_importer.cpp

```diff
@@ -257,7 +257,7 @@ struct TorchImporter
         case CV_32S:
             THFile_readIntRaw(file, (int*)storageMat.data, size);
             break;
-        case CV_USRTYPE1:
+        case CV_32SC2:
         {
             double *buf = storageMat.ptr<double>();
```

Comment:
> `.ptr<double>()`

```
int64_t *buf = storageMat.ptr<int64_t>();
```

Reply:
@alalek, The thing is that it converts int64 data to doubles. I modified parser so now we use enum variables instead OpenCV's types.